util/src/main/scala/org/ensime/util/file.scala

```diff
@@ -91,7 +90,28 @@ package object file {
      */
     @deprecating("prefer path approaches")
     def tree: Stream[File] = {
-      file #:: GFiles.fileTreeTraverser().breadthFirstTraversal(file).asScala.toStream
```

Comment:
we should definitely use NIO for this... again have a look at `path.scala` to see if we've done it already